KI-70: Add information about extra event types (#398)
centrahq/api-documentation: pages/06.plugins/15.klaviyo/docs.md

  - `Failed Subscription Payment Update`
  - `Successful Subscription Payment Update`

> Extra metrics for manual resend email from Centra, with exactly the same logic as regular ones, but requires own event type:

Reviewer: ```suggestion
Extra metrics for manual resend email from Centra, with exactly the same data as regular ones, but requires own event type:
```

> Extra metrics for manual resend email from Centra, with exactly the same data as regular ones, but requires own event type (QA only until January):

Author: ```suggestion
Extra metrics for manual resend email from Centra, with exactly the same data as regular ones, but requires own event type:
```

- `Placed Order - resend`
- `Ordered Product - resend`
- `Confirmed Order - resend`
- `Cancelled Order - resend`
- `Shipping Update - resend`
- `Refunded Order - resend`
- `Gift Certificate - resend`

> Because we have additional event types for resend, it will not have impact for your regular reports.

Reviewer: ```suggestion
Listed metrics for the email resend purposes require additional email trigger configuration for email resending to work.
Resend metrics listed above will not impact flows and reporting based on regular metrics.
```

  - `Failed Subscription Payment Update`
  - `Successful Subscription Payment Update`

Extra metrics for manual resend email from Centra, with exactly the same data as regular ones, but requires own event type:

- `Placed Order - resend`

> - `Ordered Product - resend`

Author: ```suggestion
```
